packages/io/svg-deserializer/tests/inkscape.test.js

```diff
@@ -0,0 +1,71 @@
+const test = require('ava')
+
+const deserializer = require('../index.js')
+
+// deserializer
+
+const countOf = (search, string) => {
+  let count = 0
+  let index = string.indexOf(search)
+  while (index !== -1) {
+    count++
+    index = string.indexOf(search, index + 1)
+  }
+  return count
+}
+
+test('deserialize : translate svg produced by inkscape to script', function (t) {
```

Comment:
while I admire the effort, I do not quite understand why there is a need for inkscape specific test ? are there SVG variations ?

Reply:
Seems so as there was a fix for this.

https://github.com/jscad/io/pull/72

Reply:
good point !